.github/workflows/release-please.yml

```
@@ -0,0 +1,97 @@
+on:
+  push:
+    branches:
+      - main
+
+name: release-please
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    outputs:
+      release_created: ${{ steps.release.outputs.release_created }}
+      tag_name: ${{ steps.release.outputs.tag_name }}
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - uses: google-github-actions/release-please-action@v4
```

That repo has been archived and moved to
```suggestion
      - uses: googleapis/release-please-action@v4
```

---

Refs: https://github.com/google-github-actions/release-please-action
Refs: https://github.com/googleapis/release-please-action/tree/f3969c04a4ec81d7a9aa4010d84ae6a7602f86a7/